gradle-repository/.gitattributes

```diff
@@ -0,0 +1,6 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# These are explicitly windows files and should use crlf
+*.bat           text eol=crlf
+
```

Comment:
I removed the `.gitignore`, since the files are already covered by the parent one. The `.gitattributes` file is necessary to keep the line endings for `gradlew.bat`.

Reply:
This is nit-picking, so ignore it if you want, but shouldn't the `.gitattributes` file go in the root of the repo then?